src/components/dialog-full-screen/dialog-full-screen.pw.tsx

```diff
@@ -461,6 +463,30 @@ test.describe("render DialogFullScreen component and check properties", () => {
     await dialog.press("Shift+Tab");
     await expect(select).toBeFocused();
   });
+
+  test("should render component with first input and button as focusableSelectors", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DialogFSComponentFocusableSelectors />);
+
+    const dialogElement = page.getByRole("dialog");
+    const focusedElement = page.locator("*:focus");
+    const firstInputElement = getDataElementByValue(page, "input").nth(0);
+    const secondInputElement = getDataElementByValue(page, "input").nth(1);
+    const openToastElement = getDataElementByValue(page, "open-toast");
```

Comment:
The `open-toast` data-element is for the button that opens the Toast component where as the `toastComponent` looks for the actual Toast component so in this case I don't think it would work.